blog/_posts/2016-11-29-intercontinental-gotober.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Intercontinental conference meeting @GOTOber in Berlin
+layout: post
+created_at: Tue Nov 29 2016
+permalink: blog/2016-11-29-intercontinental-gotober
+current: blog
+author: Marie & Thea, Taneea & Vrinda, Mayar & Nada
+twitter: LoadToCode
+categories:
+- student-posts
+- conferences
+---
+![RGSoC goes GOTOber](/img/blog/2016/20161128-GOTOber_1.gif)<font color="grey"><small><i>Team Twitches, Team Ruby's Secret and Team LoadToCode go GOTOber</i></small></font>
```

Comment:
This gif is awesome, but it's also incredibly huge (>11MB!!). Please compress it using one of the compression services we link to [here](https://github.com/rails-girls-summer-of-code/summer-of-code/blob/gh-pages/CONTRIBUTING.md#images).  
Same goes for images 3, 4, and 7a.